examples/widgets/Widget.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Generated On: 2016-11-22
+ * Class: Controls
+ * Description: Create a widget.
+ */
+
+function Widget(pName, pElement, options) {
```

Comment:
I might be late to the party, but you cannot really do that: the examples are neither handled by webpack nor by babel. Imho they should be seen as ready-to-use snippets of code for client apps. Therefore, they shouldn't rely on webpack, babel or anything linked to our build system. They should be valid, browser-ready javascript for our target browsers.

This actually means that:
- we need to formally decide which our target browsers (and versions) are (because it defines which js api/functions/syntax we can safely use)
- we need a special `.eslintrc` for the `examples` folder